Sources/StreamChatUI/Navigation/ChatMessageListRouter.swift

```diff
@@ -55,6 +54,52 @@ open class ChatMessageListRouter:
         rootViewController.present(popup, animated: true)
     }
 
+    /// Shows the detail pop-up for the selected message with all the message reactions presented.
+    ///
+    /// - Parameters:
+    ///   - messageContentView: The selected message content view.
+    ///   - client: The current `ChatClient` instance.
+    open func showReactionsPopUp(
+        messageContentView: ChatMessageContentView,
+        client: ChatClient
```

Comment:
nit: the `ChatMessageListRouter` was initially used on both channel and thread controllers so it was not possible to make it generic over a single VC type so we went with `NavigationRouter<UIViewController>`. But now with the separated message list it can be `NavigationRouter<ChatMessageListVC>` so there'll be no need to explicitly pass the client 👍

Reply:
Hum, but this might limit the customer right? 🤔 Do we want to do that? @evsaev And this might even be a breaking change, if a customer is using this router without the message list some how 🤔